src/content/getting-started/architecture/globalnet/_index.en.md

```diff
@@ -78,10 +91,78 @@ Globalnet currently relies on `kube-proxy` and thus will only work with deployme
 Connectivity is only part of the solution as pods still need to know the IPs of services on remote clusters.
 
 This is achieved by enhancing [lighthouse](https://github.com/submariner-io/lighthouse) with support for Globalnet. The Lighthouse
-controller adds the service's globalIp to the `ServiceImport` object that is distributed to all clusters. The [lighthouse
-plugin](https://github.com/submariner-io/lighthouse/tree/devel/plugin/lighthouse) then uses the Service's globalIp when replying to DNS
-queries for the Service.
+controller uses pod's globalIp instead of podIp when creating `EndpointSlice` objects to be distributed to other clusters.
+The [lighthouse plugin](https://github.com/submariner-io/lighthouse/tree/devel/plugin/lighthouse) then uses the Service's globalIp when
+replying to DNS queries for the Service.
 
 ## Building
 
 Nothing extra needs to be done to build `submariner-globalnet` as it is built with the standard Submariner build.
+
+## Usage
+
+Refer [Quickstart Guides](../../quickstart/) on how to deploy Submariner with Globalnet enabled. For most deployments users will not
+need to do anything else once deployed. But users can create GlobalEgressIPs or edit ClusterGlobalEgressIP for specific use cases.
+
+### Ephemeral Port Exhaustion
+
+By default `8` globalIps are allocated as ClusterGlobalEgressIPs. This allows for `~8x64k` active ephemeral ports. If those are still
+not enough for a cluster, users can increase the number of ClusterGlobalIPs allocated. This will require a restart/re-join of the cluster.
+Note that this assumes that there are still some globalIps available in the cluster pool. To increase the number of globalIps allocated
+for ClusterGlobalEgressIPs, create the following object with `NumberOfIPs` set to required value.
+
+```yaml
+   apiVersion: submariner.io/v1alpha1
+   kind: ClusterGlobalEgressIP
+   metadata:
+     name: cluster-egress.submariner.io
+   spec:
+     NumberOfIPs: 8
+```
+
+{{% notice note %}}
+Name must be `cluster-egress.submariner.io`. Any different name will be ignored by globalnet and appropriate `Status` will be set.
+`NumberOfIPs` can have minimum value of `0` and maximum of `20`.
+{{% /notice %}}
+
+### GlobalIp for a Namespace
+
+For use cases where user wants all pods in a namespace to use a unique globalIp instead of one of the ClusterGlobalEgressIPs,
+users can create
+GlobalEgressIP for that namespace.
```

Comment:
```suggestion
If it's desired for all pods in a namespace to use unique global IP(s) instead of one of the cluster-scoped IPs, a user can create a `GlobalEgressIP` resource in that namespace:
```